ui/js/component/app/view.jsx

```diff
@@ -11,12 +11,18 @@ class App extends React.PureComponent {
       checkUpgradeAvailable,
       updateBalance,
       fetchRewardedContent,
+      getThemes,
+      setTheme,
     } = this.props;
 
     document.addEventListener("unhandledError", event => {
       alertError(event.detail);
     });
 
+    getThemes();
+
+    setTheme();
```

Comment:
I think getThemes would only need to be called by the `willComponentMount` method in `page/settings` and we should use a design that doesn't require a call to `setTheme` at all.